test/test_entanglement_tracker_grid.jl

```diff
@@ -7,145 +7,193 @@ using QuantumSavory.ProtocolZoo: EntanglementCounterpart, EntanglementHistory, E
 using Graphs
 using Test
 
-if isinteractive()
-    using Logging
-    logger = ConsoleLogger(Logging.Debug; meta_formatter=(args...)->(:black,"",""))
-    global_logger(logger)
-    println("Logger set to debug")
-end
-
 ##
```

Comment:
I do not really follow what is being tested here. Could you describe in words what each section is doing?